xml/docu_styleguide.language.xml

```diff
@@ -271,12 +273,12 @@
     </listitem>
     <listitem>
       <para>
-	If a list entry consists of a single sentence, do not use period
+ If a list entry consists of a single sentence, do not use a period
```

Comment:
This section about lists (largely) duplicates content that is also part of the Structure section. As such, it makes more sense to delete it rather than fix the language issues here. :)

Reply:
OK